tasks/src/text-to-image/data.ts

```diff
@@ -26,7 +26,23 @@ const taskData: TaskDataCustom = {
 			},
 		],
 	},
-	metrics: [],
+	metrics: [
+		{	description: " The underlying concept of the Inception Score (IS) as a metric is to assess the quality of a model based on its ability to generate diverse and meaningful images. This is achieved by using a generated image sample x to predict its label y using the Inception model. 
```

Comment:
Is it okay if we could make this shorter? 😊

Reply:
made it shorter :)